cmake/FindDPP.cmake

```diff
@@ -1,15 +1,41 @@
-if(WIN32 AND NOT EXISTS ${CMAKE_SOURCE_DIR}/deps/dpp.lib)
-string(TOLOWER ${CMAKE_BUILD_TYPE} INSTALL_DPP_BUILD_TYPE)
-execute_process(COMMAND powershell "-NoLogo" "-NoProfile" "-File" ".\\install_dpp_msvc.ps1" ${INSTALL_DPP_BUILD_TYPE} WORKING_DIRECTORY ${CMAKE_SOURCE_DIR})
```

Comment:
The `install_dpp_msvc.ps1` script here automatically installs the D++ library for Microsoft Visual C++ users. Why remove it?

Reply:
PS: I removed every DPP-related installation scripts because I cannot and don't know how to build it using that on Windows. It shows too much build errors when I first ran it, I don't know if I did something wrong or something. That was the first time I cloned the repository btw

Reply:
Huh? but it worked for me...

Reply:
Can you show me what the errors are?